FaunaDB.Client/Types/RefV.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using FaunaDB.Query;
+using Newtonsoft.Json;
+
+namespace FaunaDB.Types
+{
+    /// <summary>
+    /// A FaunaDB ref type.
+    /// <para>
+    /// See <see href="https://fauna.com/documentation/queries#values-special_types">FaunaDB Special Types</see>.
+    /// </para>
+    /// </summary>
+    public class RefV : Value
+    {
+        public string Id { get; }
+        public RefV Class { get; }
+        public RefV Database { get; }
```

Comment:
In theory those are `ClassV` and `DatabaseV` but in reality, those are any reference that the server returns. I doesn't make sense to restrict the type, IMO.